JavaScript/JavaScriptSDK/ajax/ajax.ts

```diff
@@ -140,9 +140,9 @@ module Microsoft.ApplicationInsights {
         private sendHandler(xhr: XMLHttpRequestInstrumented, content) {
             xhr.ajaxData.requestSentTime = dateTime.Now();
 
-            // Add correlation headers only for requests within the same domain (ignore the port number)
+            // Add correlation headers only for requests within the same host and port number
```

Comment:
I do not see that port is added? Either description is misleading or the fix was not fully implemented. Can you please double check?